includes/Login.php

```diff
@@ -6,10 +6,15 @@
 
 if( isset( $_SESSION['Username'] ) )
 {
+	
 	if( $_SESSION['Username'] == 'admin' )
 	{
 		header("Location:includes/dashboard/Admin.php");
 	}
+	else if( isset( $_SESSION['PostName'] ) )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Usage of ELSE IF is discouraged; use ELSEIF instead
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

<!-- Comment by [SideCI](https://sideci.com) -->